.travis.yml

```diff
@@ -4,12 +4,14 @@ dist: trusty
 sudo: required
 language: python
 
-python:
-  - 2.7
-  - 3.7-dev
-  - 3.6
-  - 3.5
-  - 3.4
+matrix:
+  include:
+    - python: 2.7
+    - python: 3.7
+      dist: xenial
```

Comment:
Looks like xenial doesn't support postgres 9.3 (and 9.2 I guess): I can help building them. If possible I'd move the entire grid on xenial.

Reply:
> If possible I'd move the entire grid on xenial.

Sure thing. This will work for all versions except 3.4, which is not available on xenial.